todo/src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import reducers from './reducers';
+
+import './index.css';
+import App from './App';
+
+ReactDOM.render(
+    <Provider store={createStore(reducers)}>
```

Comment:
👍 You may also consider breaking the store=createstore out of provider, and making a cost above you can reference in provider,  You can also pass in your redux dev tools extension there.